cockatrice/src/tab_supervisor.cpp

```diff
@@ -559,3 +561,19 @@ bool TabSupervisor::getAdminLocked() const
         return true;
     return tabAdmin->getLocked();
 }
+
+void TabSupervisor::processNotifyUserEvent(const Event_NotifyUser &event)
+{
+    QString notificationTitle("Notification");
+    QString notificationMessage;
+    switch ((Event_NotifyUser::NotificationType) event.type()) {
+        case Event_NotifyUser::PROMOTIONSUCCESS: QMessageBox::information(this, tr("Success"), tr("Successfully promoted user.")); break;
```

Comment:
The reason it was written this way was to allow for the ability to be able to notify all staff members of the event (should we decide to change the code to do so).